src/view/dataElements/components/javascriptTools/startEndPosition.jsx

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React from 'react';
+import { TextField, Flex } from '@adobe/react-spectrum';
+import WrappedField from '../../../components/wrappedField';
+import { isNumberLike } from '../../../utils/validators';
+import operators from '../../utils/javascriptTools/operators';
+
+export default () => (
+  <Flex gap="size-100">
+    <WrappedField
```

Comment:
`type="number"`?

Reply:
We never used it in other places where we were requesting the user to type numbers. One disadvantage is that nothing happens when you type in a number field field something else than a number. It might cause the impression that there is something wrong with the app.